.github/workflows/generate-desktop-targets.yml

```diff
@@ -57,8 +59,32 @@ jobs:
           FLEET_DESKTOP_NOTARIZE=true \
           FLEET_DESKTOP_VERSION=$FLEET_DESKTOP_VERSION \
           make desktop-app-tar-gz
+
       - name: Upload desktop.app.tar.gz
         uses: actions/upload-artifact@82c141cc518b40d92cc801eee768e7aafc9c2fa2 # v2
         with:
           name: desktop.app.tar.gz
           path: desktop.app.tar.gz
+
+  desktop-windows:
+    runs-on: macos-latest
```

Comment:
Do we have to run this on mac? It might be faster on a linux container.

Reply:
Good point, will change it when we implement Linux desktop.